### DISCUSSION TITLE
Fieldwork 2021

### DISCUSSION THREAD

**Discussion by bscelza** [endowproject/endowproject.github.io#6] [Q&A]
Just wanted to get the conversation started about what people are thinking about fieldwork for the ENDOW project this summer. Anybody have definite plans to go or not to go?

**Comment by bscelza** [comment]
Apparently there is another thread on this somewhere that I can't find--so I guess just ignore this and keep going on the current discussion! Sorry for the duplication.

**Reply by eapower** [reply]
Convo is over here, Brooke! --> https://github.com/danielRedhead/endow-resources/issues/12

**Comment by bscelza** [comment]
Thanks. I’ve been emailing with Jeremy but I can’t access it. Even if I’m
logged in I just get an error screen....

On Tue, Mar 30, 2021 at 12:46 AM Eleanor Power ***@***.***>
wrote:

> Convo is over here, Brooke! --> danielRedhead/endow-resources#12
> <https://github.com/danielRedhead/endow-resources/issues/12>
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/endowproject/endowproject.github.io/discussions/6#discussioncomment-546958>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ATORIQTPJZVM6O54YGKTKOLTGF6W7ANCNFSM42A2MLFA>
> .
>
-- 
Brooke A. Scelza
Associate Professor
UCLA Department of Anthropology
341 Haines Hall, Box 951553
Los Angeles, CA 90095-1553
***@***.***

*The Department of Anthropology at UCLA acknowledges the Gabrielino/Tongva
peoples as the traditional land caretakers of Tovaangar (the Los Angeles
basin and So. Channel Islands). As a land grant institution, we pay our
respects to the Honuukvetam (Ancestors), ‘Ahiihirom (Elders) and
‘Eyoohiinkem (our relatives/relations) past, present and emerging.*

**Reply by eapower** [reply]
You need to be logged in to GitHub and have access to the private repo (you should be getting an invite imminently from Dan/GitHub).

**Reply by danielRedhead** [reply]
Hey Brooke, you should've just received an invite to the repo. Please accept the invitation within 7 days otherwise it'll lapse! Let us know if you encounter any issues!
